docs/user-guide/operations/deploy-on-kubernetes/deploy-greptimedb-cluster.md

```diff
@@ -1,44 +1,105 @@
+```
+
+## Deploy a minimum GreptimeDB cluster
 
-Deploy the GreptimeDB cluster, ensuring it connects to the previously established etcd cluster:
-  
+Next, deploy a minimum GreptimeDB cluster. The deployment will depend on an etcd cluster for coordination. If you already have one running, you can use its endpoint. If you followed the steps above, use `etcd.etcd-cluster.svc.cluster.local:2379` as the etcd endpoint.
+
+Run this command to install the GreptimeDB cluster:
 ```shell
 helm install greptimedb greptime/greptimedb-cluster \
   --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
   --create-namespace \
   -n greptimedb-cluster
 ```
 
-### Setting Resource requests and limits
+### Verify the GreptimeDB cluster installation
+Check that all pods are running properly in the `greptimedb-cluster` namespace:
+
+```bash
+kubectl get pods -n greptimedb-cluster
+```
+
+You should see output similar to this:
+```bash
+NAME                                      READY   STATUS    RESTARTS   AGE
+my-greptimedb-datanode-0                  1/1     Running   0          30s
+my-greptimedb-datanode-2                  1/1     Running   0          30s
+my-greptimedb-datanode-1                  1/1     Running   0          30s
+my-greptimedb-frontend-7476dcf45f-tw7mx   1/1     Running   0          16s
+my-greptimedb-meta-689cb867cd-cwsr5       1/1     Running   0          31s
+```
+
+## Advanced Configuration Options
 
-The GreptimeDB Helm charts enable you to specify resource requests and limits for each component within your deployment.
+### Use Object Storage as backend storage
+To store data on Object Storage (here is example to store on Amazon S3), add the following configuration to your Helm command:
+
+```bash
+helm install \
+  --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
+  --set objectStorage.s3.bucket="your-bucket" \
+  --set objectStorage.s3.region="region-of-bucket" \
+  --set objectStorage.s3.root="root-directory-of-data" \
+  --set objectStorage.credentials.accessKeyId="your-access-key-id" \
+  --set objectStorage.credentials.secretAccessKey="your-secret-access-key" \
+  greptime/greptimedb-cluster \
+  -n greptimedb-cluster
+```
 
-Here's how you can configure these settings:
+### Using RemoteWAL and enable the Region Failover
+If you want to enable RemoteWAL and region failover, follow this configuration. You’ll need a Kafka cluster running, and you can use its endpoint like `kafka.kafka-cluster.svc.cluster.local:9092`:
+
+```bash
+helm install \
+  --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
+  --set meta.enableRegionFailover=true \ 
+  --set objectStorage.s3.bucket="your-bucket" \
+  --set objectStorage.s3.region="region-of-bucket" \
+  --set objectStorage.s3.root="root-directory-of-data" \
+  --set objectStorage.credentials.accessKeyId="your-access-key-id" \
+  --set objectStorage.credentials.secretAccessKey="your-secret-access-key" \
+  --set remoteWal.enable=true \
```

Comment:
After this PR is merged, the installation command will change. It is recommended to try it after merging and then modify the installation command.
https://github.com/GreptimeTeam/greptimedb-operator/pull/180